pkg/nfd-master/node-updater-pool.go

```diff
@@ -55,6 +55,7 @@ func (u *nodeUpdaterPool) processNodeLabelRequest(queue workqueue.RateLimitingIn
 			klog.ErrorS(err, "error labeling node", "nodeName", nodeName)
 		}
 	}
+	updatedNodes.Inc()
```

Comment:
I think this will cause double accounting. We have this in `refreshNodeFeatures()` and that should be enough (as we will end up there from this method)